acceptance-tests/src/test/java/tech/pegasys/pantheon/tests/acceptance/dsl/node/ThreadPantheonNodeRunner.java

```diff
@@ -154,6 +154,10 @@ public void startNode(final PantheonNode node) {
             .metricsConfiguration(node.metricsConfiguration())
             .p2pEnabled(node.isP2pEnabled())
             .graphQLConfiguration(GraphQLConfiguration.createDefault())
+            .staticNodes(
+                node.getStaticNodes().stream()
```

Comment:
this has a potential of NPE unless we make sure getStaticNodes will never return null.

Reply:
We always initialize it in `PantheonFactoryConfigurationBuilder` as an empty list. There is a lot of indirection to get there but we should be safe for now.